examples/models-as-a-service-3scale/3scale/values-apis.yaml

```diff
@@ -5,10 +5,11 @@
 
 # https://github.com/3scale/3scale-operator/blob/master/doc/product-reference.md
 apis:
-  - name: granite-8b-code-instruct
-    activeDocOpenAPIRef: https://raw.githubusercontent.com/redhat-gpte-devopsautomation/llm-aas/refs/heads/main/bootstrap/3scale/api_definitions/granite-8b-code-instruct.json
+  - name: llama-32-1b-instruct-cpu
+    # activeDocOpenAPIUrl: https://raw.githubusercontent.com/redhat-ai-services/ai-accelerator-examples/refs/heads/mlaas-gitops/examples/models-as-a-service-3scale/3scale-config/api_definitions/llama-32-1b-instruct.json
```

Comment:
Added the option to bundle the OpenAPI definition instead of having it referenced on an external environment. I left the original implementation commented here.